Formula/pcre2.rb

```diff
@@ -1,29 +1,12 @@
 class Pcre2 < Formula
   desc "Perl compatible regular expressions library with a new API"
   homepage "https://www.pcre.org/"
+  url "https://github.com/PhilipHazel/pcre2/releases/download/pcre2-10.40/pcre2-10.40.tar.bz2"
+  sha256 "14e4b83c4783933dc17e964318e6324f7cae1bc75d8f3c79bc6969f00c159d68"
   license "BSD-3-Clause"
 
-  # Remove `stable` block next release when patches are no longer needed
-  stable do
-    url "https://github.com/PhilipHazel/pcre2/releases/download/pcre2-10.39/pcre2-10.39.tar.bz2"
-    sha256 "0f03caf57f81d9ff362ac28cd389c055ec2bf0678d277349a1a4bee00ad6d440"
-
-    # enable JIT again in Apple Silicon with 11.2+ (sljit PR zherczeg/sljit#105)
-    patch :p2 do
-      url "https://github.com/zherczeg/sljit/commit/d6a0fa61e09266ad2e36d8ccd56f775e37b749e9.patch?full_index=1"
-      sha256 "8d699f6c8ae085f50cf8823dcfadb8591f7ad8f9aa0db9666bd126bb625d7543"
-      directory "src/sljit"
-    end
-
-    # https://lists.gnu.org/archive/html/libtool-patches/2020-06/msg00001.html
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-      sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-    end
-  end
-
   livecheck do
-    url :stable
+    url :url
```

Comment:
The original is still correct. Just because there isn't an explicit stable block doesn't mean `:stable` doesn't work.

Reply:
I can do a syntax only PR after this is merged if the preference is to set as `:stable`

Reply:
It's not really a preference thing. `:url` is Cask-specific and does not work on formula.